modules/exploits/linux/local/servu_ftp_server_prepareinstallation_priv_esc.rb

```diff
@@ -108,32 +108,26 @@ def upload_and_chmodx(path, data)
   end
 
   def check
-    unless command_exists? 'bash'
-      vprint_error 'bash shell is not available'
-      return CheckCode::Safe
-    end
+    return CheckCode::Safe('bash shell is not available') unless command_exists? 'bash'
+
     vprint_good 'bash shell is available'
 
-    unless cmd_exec("test -x '#{servu_path}' && echo true").include? 'true'
-      vprint_error "#{servu_path} is not executable"
-      return CheckCode::Safe
-    end
+    return CheckCode::Safe("#{servu_path} file not found") unless file? servu_path
+
+    return CheckCode::Safe("#{servu_path} is not executable") unless cmd_exec("test -x '#{servu_path}' && echo true").include? 'true'
```

Comment:
An `executable?` method was added for this purpose after this module was written.

https://github.com/rapid7/metasploit-framework/blob/c2e593124827c045a46bf5d8338d251e09c3cea5/lib/msf/core/post/file.rb#L234-L238

```suggestion
    return CheckCode::Safe("#{servu_path} is not executable") unless executable?(servu_path)
```